app/_locales/am/messages.json

```diff
@@ -1125,13 +1125,16 @@
     "message": "ዕውቂያን ይመልከቱ"
   },
   "viewOnCustomBlockExplorer": {
-    "message": "በ $1ይመልከቱ"
+    "message": "በ $1ይመልከቱ",
```

Comment:
To clarify, what I meant is that these three messages should be removed from non-English locales completely. Right now the wallet will throw an error when trying to display these messages because they're expected to have a `$1` in them, but there is no `$1`.

The easiest way to delete them all is to rename the English messages, run `yarn verify-locales:fix` (which will delete them in each other locale), then rename the English ones back to the original names.

Reply:
I see, I ran :fix without renaming first.